### PR TITLE
refactor function name isIPAssignedToPod to isIPAssignedToOtherPod

### PR DESCRIPTION
**Pull request by changluyi**
- [ ] Make sure you have followed [Kube-OVN Code Style](https://github.com/kubeovn/kube-ovn/blob/master/CODE_STYLE.md).

#### What type of this PR
Examples of user facing changes:
- Features
- Bug fixes
- Docs
- Tests

#### Which issue(s) this PR fixes:
Fixes #(issue-number)

refactor function name isIPAssignedToPod to isIPAssignedToOtherPod

### CODE REVIEW

**Review by hongzhen-ma — APPROVED** [review]

File: pkg/controller/pod.go [hunk 1]
```diff
@@ -1297,7 +1297,7 @@ func (c *Controller) acquireAddress(pod *v1.Pod, podNet *kubeovnNet) (string, st
 		for _, net := range nsNets {
 			for _, staticIPs := range ipPool {
 				for _, staticIP := range strings.Split(staticIPs, ",") {
-					if c.ipam.IsIPAssignedToPod(staticIP, net.Subnet.Name, key) {
+					if c.ipam.IsIPAssignedToOtherPod(staticIP, net.Subnet.Name, key) {
 						klog.Errorf("static address %s for %s has been assigned", staticIP, key)
 						continue
```
Comment (zhangzujian) [inline pkg/controller/pod.go:1302]: Can we provide information about the pod to which the IP is assigned? This is helpful for debugging.

Comment (changluyi) [reply pkg/controller/pod.go:1302]: got it

Comment (changluyi) [reply pkg/controller/pod.go:1302]: subnet.isIPAssignedToOtherPod provide the log:

		klog.V(4).Infof("v4 check ip assigned, existPod %s, podName %s", existPod, podName)

existedPod is  the pod to which the IP is assigned

Comment (zhangzujian) [reply pkg/controller/pod.go:1302]: `klog.V(4).Infof` means the message will be printed only when the logging verbosity level is greater than or equal to 4.

To change the verbosity level from the default value to 4, we need to add an additional command argument`-v=4` to kube-ovn-controller.

Comment (changluyi) [reply pkg/controller/pod.go:1302]: Do you mean below?

klog.V(4).Infof  => klog.Infof

Comment (zhangzujian) [reply pkg/controller/pod.go:1302]: I'd prefer this:

```go
if assignedPod, ok := c.ipam.IsIPAssignedToOtherPod(staticIP, net.Subnet.Name, key); ok {
    klog.Errorf("static address %s for %s has been assigned to %s", staticIP, key, assignedPod)
    // ...
}
```

Comment (changluyi) [reply pkg/controller/pod.go]: ok, I push my new pr

File: pkg/ipam/ipam.go [hunk 2]
```diff
@@ -242,14 +242,14 @@ func (ipam *IPAM) ContainAddress(address string) bool {
 	return false
 }
 
-func (ipam *IPAM) IsIPAssignedToPod(ip, subnetName, podName string) bool {
+func (ipam *IPAM) IsIPAssignedToOtherPod(ip, subnetName, podName string) (bool, string) {
```
Comment (zhangzujian) [inline pkg/ipam/ipam.go:245]: The return values should be in order `(string, bool)`.

**Review by zhangzujian — APPROVED** [review]

**Pull request merged**